README.md

```diff
@@ -111,3 +111,29 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+Run:
+```Shell
+tools/setup_env.sh
```

Comment:
I agree that git and git-secrets feels like a dev host machine concern. 
The other two questions might warrant asking in the slack or the docker PR?